flow/include/flow/genericactors.actor.h

```diff
@@ -1848,13 +1850,18 @@ class AndFuture {
 	}
 
 	void add(Future<Void> const& f) {
+		began_ = true;
 		if (!f.isReady() || f.isError())
 			futures.push_back(f);
 	}
 
 	void add(AndFuture f) { add(f.getFuture()); }
 
+	// Whether or not there has ever been a future associated with this AndFuture
+	bool began() const { return began_; }
+
 private:
+	bool began_ = false;
```

Comment:
What is the stylistic significance of the trailing `_`?

Reply:
I get that there would be a name collision with `began()` but a more common pattern in the FDB code base is to have a member var `foo` and an accessor `getFoo()`.

Reply:
What `began` seems to be really tracking is whether or not the number of futures added to this object is greater than 0.  Perhaps it would be better to just track this count as a `count` member and add `getCount()`.

Similarly though, a count method could be added to ActorCollection and think it may be a better choice here.   Since the recent refactor of it it shouldn't be slower anymore and it has the benefit of dropping future references as they complete, though I'm not sure if that matters in the RYW cache.

Reply:
Added `getFutureCount`